bin/node/cli/src/chain_spec.rs

```diff
@@ -64,7 +64,7 @@ pub fn pangolin_session_keys(
 pub fn properties() -> Properties {
 	let mut properties = Properties::new();
 
-	properties.insert("ss58Format".into(), 42.into());
+	properties.insert("ss58Format".into(), 18.into());
```

Comment:
Are we going to change Pangolin ss format to 18?

Reply:
Yes, already live in pangolin-rpc.darwinia.network

@AsceticBear @WoeOm want to make keep same with mainnet which good for testing.